assets/js/details.js

```diff
@@ -0,0 +1,23 @@
+  const params = new URLSearchParams(window.location.search);
+  const movieId = params.get("id");
+  const apiKey = "9f120f0e95e5cedf05a78abda9ef8477";
+  const detailsContainer = document.getElementById("details-container");
+  
+  async function getMovieDetails () {
+   const result = await fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}`);
+   const {poster_path: img, title, vote_average, overview} = await result.json();
```

Comment:
Rename the variables in a camel case format while destructuring.

Reply:
Done.